### DISCUSSION TITLE
计划接入AI绘图吗

### DISCUSSION THREAD

**Discussion by xueandyue** [ChatGPTNextWeb/NextChat#1463] [Ideas]
大佬有计划接入AI绘图功能吗？看到其他国内封装chatgpt相关的应用，有些能做AI绘图，也不懂是怎么搞的？midjourney 也没提供api

**Comment by leochen12-rgb** [comment]
有现成的好像，叫chatgptWithMidjourney

**Reply by eric-easyhealth** [reply]
这是个插件么？

**Reply by lloydzhou** [reply]
1. 现在的主版本支持sd绘图
2. 商业版支持mj绘图
3. 另外v2.15.0支持插件，可以将绘图api包装成插件在会话的时候调用
